spec/app/models/job_offer_spec.rb

```diff
@@ -193,4 +193,31 @@
     end
 
   end
+
+  describe 'deactivate owners offers' do
```

Comment:
Me parece que sería mejor como "the user doesn't get his offers to apply" o "avoid the user from applying to his own offers"